app/frontend/Program.cs

```diff
@@ -18,6 +20,12 @@
 builder.Services.AddSpeechRecognitionServices();
 builder.Services.AddMudServices();
 
+builder.Services.AddSingleton<ILocalStorageServiceWrapper, LocalStorageServiceImplementation>();
```

Comment:
For anyone following along, this is the part that isn't so pretty. For now there is an additional service interface layer to abstract the WebAssembly services out so that we can have a different implementation for .NET MAUI.

I have some thoughts on how to improve this, but I have to check if my ideas will work.

Reply:
This is gone now.